.github/workflows/functional-baremetal.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: Checkout Gophercloud
         uses: actions/checkout@v4
       - name: Deploy devstack
-        uses: EmilienM/devstack-action@v0.11
+        uses: EmilienM/devstack-action@v0.12
```

Comment:
For anyone looking, the fix consists in getting the ubuntu packages at their latest version prior to installing devstack.
The proposed fix in devstack-action is at https://github.com/EmilienM/devstack-action/pull/15.